jobbergate-cli/jobbergate_cli/subapps/job_submissions/tools.py

```diff
@@ -74,7 +74,7 @@ def __post_init__(self):
         self.cluster_name = _map_cluster_name(self.jg_ctx, self.cluster_name)
 
         if self.execution_directory is None:
-            self.execution_directory = Path.cwd()
+            self.execution_directory = Path.home()
```

Comment:
The change on the agent side looks good, but there is no need to change it on the cli, `cwd` is the expected here.

Reply:
Addressed by [5b2c67c](https://github.com/omnivector-solutions/jobbergate/pull/732/commits/5b2c67cc36f1921a61b226570ea1a3275a0d414c).